debian/xivo-certs.postinst

```diff
@@ -10,10 +10,10 @@ case "$1" in
         # generate X.509 certificates if necessary. Valid for 10 years.
```

Comment:
Does that comment also need to be modified?

Reply:
Very true, thanks :) Fixed